droid-results/src/main/java/uk/gov/nationalarchives/droid/profile/ProfileContextLocator.java

```diff
@@ -432,7 +432,8 @@ private ProfileInstance createNewProfileInstance(String id, PropertiesConfigurat
         profileInstance.setColumnsToWrite(mergedConfig.getString(DroidGlobalProperty.COLUMNS_TO_WRITE.getName(), ""));
         profileInstance.setExportOptions(ExportOptions.valueOf(mergedConfig.getString(DroidGlobalProperty.EXPORT_OPTIONS.getName(),
                 ExportOptions.ONE_ROW_PER_FILE.name())));
-        if (mergedConfig.containsKey(DroidGlobalProperty.UPDATE_USE_PROXY.getName()) && mergedConfig.getBoolean(DroidGlobalProperty.UPDATE_USE_PROXY.getName())) {
+        String updateUseProxyName = DroidGlobalProperty.UPDATE_USE_PROXY.getName();
+        if (mergedConfig.containsKey(updateUseProxyName) && mergedConfig.getBoolean(updateUseProxyName)) {
             URI proxy = URI.create("http://" + mergedConfig.getString(DroidGlobalProperty.UPDATE_PROXY_HOST.getName()) + ":" + mergedConfig.getInt(DroidGlobalProperty.UPDATE_PROXY_PORT.getName()));
```

Comment:
There's still these 2 that haven't been replaced

Reply:
The UPDATE_PROXY_HOST and UPDATE_PROXY_PORT ones? They don't need to be, they're only used once. A variable is ok for UPDATE_USE_PROXY as it's used twice but I don't want interim variables everywhere.